src/pages/UsedCardScreen.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Prose from '../components/Prose'
+import Main, { MainChild } from '../components/Main'
+
+const ExpiredCardScreen = () => (
```

Comment:
copy-pasta

```suggestion
const UsedCardScreen = () => (
```

Reply:
TSLint had a rule to ensure that default export was named same as filename. I guess I got lazy after relying on it. 😛  Fixing.